src/lib/behavior.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (obj) => {
+  return (eventName, data, func, opt) => {
+    var hiddenCancelled = false;
+    var cancelled = false;
+    var cancel = (hidden) => { // Hidden shouldn't be used often but it's not hard to implement so meh
```

Comment:
It cancels the behavior without setting "cancelled" to true. This means you can trick other plugins into doing things. Really easy to implement for now, so if we don't find any use cases we can remove it.